content/en/news/releases/1.18.x/announcing-1.18/change-notes/index.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Istio 1.18.0 Change Notes
+linktitle: 1.18.0
+subtitle: Minor Release
+description: Istio 1.18.0 change notes.
+publishdate: 2023-06-03
+release: 1.18.0
+weight: 20
+---
+
+
+# Change Notes
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/about/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+
+- There are no new deprecations in Istio 1.18.0
+
+## Traffic Management
+
```

Comment:
I know these aren't done yet, but noting that some Actions aren't all together within a group.

Reply:
As noted in channel, it's not necessary to fix the grouping as it's done automatically at build time.  I will fix the whitespace though so linting passes